framework_tvb/tvb/interfaces/web/controllers/project/project_controller.py

```diff
@@ -677,14 +677,15 @@ def updatemetadata(self, **data):
     @cherrypy.expose
     @handle_error(redirect=False)
     @check_user
-    def downloaddatatype(self, data_gid, export_module):
+    def downloaddatatype(self, data_gid, export_module, **data):
         """ Export the data to a default path of TVB_STORAGE/PROJECTS/project_name """
         current_prj = common.get_current_project()
         # Load data by GID
         entity = load_entity_by_gid(data_gid)
         # Do real export
         export_mng = ExportManager()
-        file_name, file_path, delete_file = export_mng.export_data(entity, export_module, current_prj)
+        file_name, file_path, delete_file = export_mng.export_data(entity, export_module, current_prj,
+                                                                   data['exporter_key'])
```

Comment:
Use `data.get('exporter_key')` to avoid inexistent key error and get None value instead